Backend/TimeTracker.Test/ProfileTests.cs

```diff
@@ -0,0 +1,148 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;  // TestClass
+using TimeTracker.Api.Controllers;                   // ProfileController
+using TimeTracker.Api.Helpers;                       // AuthHelper
+using TimeTracker.Api.DTOs;                          // DTOs
+using System.Threading.Tasks;                        // Task
+using System.Security.Claims;                        // Claims stuff
+using System.Collections.Generic;                    // List
+using Microsoft.AspNetCore.Http;                     // DefaultHttpContext
+using Microsoft.EntityFrameworkCore;                 // .AsNoTracking()
+using TimeTracker.Api.Database.Models;               // User
+using System.Linq;                                   // SequenceEqual
+
+namespace TimeTracker.Test {
+    [TestClass]
+    public class ProfileTests : BaseTest {
+        ProfileController profileController;
+        AuthController authController;
+        AuthHelper authHelper;
+        public ProfileTests() {
+            authHelper = new AuthHelper();
+            profileController = new ProfileController(database, authHelper, configuration);
+            authController = new AuthController(database, configuration, authHelper);
+        }
+
+        private void attachUserToContext(int userID) {
```

Comment:
Nice work getting this figured out, I made a helper class to do this sadly (maybe I should ping in the discord or mention it in our meetings) TestAuthHelpers.LogInUser, may look better if you use that function here.

Reply:
I see you are creating users and such in the tests though, so maybe that doesn't work the best in your case here. If you want you could move this function to the helper's class as well because I am sure it will be useful in other tests.